testing/visual-regression/backstop-common.js

```diff
@@ -1,8 +1,37 @@
-module.exports = function (baseUrl) {
+module.exports = function backstopCommon(baseUrl) {
   function storyUrlFor(componentId) {
     return `${baseUrl}/iframe.html?id=${componentId}&viewMode=story`;
   }
 
+  function buildButtonScenarios(labelPrefix, componentId) {
+    const viewport = { label: 'button', width: 200, height: 100 };
+    return [
+      {
+        label: `${labelPrefix}`,
+        url: storyUrlFor(componentId),
+        viewports: [viewport],
+      },
+      {
+        label: `${labelPrefix} (hover)`,
+        url: storyUrlFor(componentId),
+        hoverSelector: 'button',
```

Comment:
will this scope correctly to the right button? I know we're passing the selector in as well, shouldn't we use that?

Reply:
Yep, with the new docs formats the _only_ thing in the scenario is the one button. So this is intentional.

Reply:
Rightio, that's quite cool. Makes it a bit easier.